rubycritic.gemspec

```diff
@@ -35,30 +35,30 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'launchy', '>= 2.5.2'
   spec.add_dependency 'parser', '>= 3.3.0.5'
   spec.add_dependency 'rainbow', '~> 3.1.1'
-  spec.add_dependency 'reek', '~> 6.4.0', '< 7.0'
+  spec.add_dependency 'reek', '~> 6.5.0', '< 7.0'
```

Comment:
This is the most significant change (not a dev dependency): https://github.com/troessner/reek/blob/master/CHANGELOG.md

But based on the reek changelog it should be fine to merge.